src/Individual_XP.cpp

```diff
@@ -87,6 +116,34 @@ class Individual_XP_command : public CommandScript
         me->GetSession()->SendAreaTriggerMessage("You have updated your XP rate to %u", rate);
         return true;
     }
+    
+    static bool HandleDisableCommand(ChatHandler* handler, char const* args)
+    {
+        if (*args)
+            return false;
+          
+        Player* me = handler->GetSession()->GetPlayer();
+        if (!me)
+            return false;
+        
+        me->CustomData.GetDefault<PlayerXpRate>("Individual_XP")->XPRate = 0;
+        me->GetSession()->SendAreaTriggerMessage("You have updated your XP rate to 0");
```

Comment:
Could change the message to "You have disabled XP gain. (XP rate is set to 0)"

Talamortis wanted to add the option to disable XP gain differently, completly disabling XP gain instead of setting the rate to 0 because when rate=0 then every mob killed will display "0 xp gain" or something like that

Reply:
I could look into doing something like that...